adserver/templates/adserver/reports/all-publishers.html

```diff
@@ -40,12 +40,14 @@
 {% block toc %}
 <section class="mb-5">
   {% if publishers %}
-    <h2>{% trans 'Publishers '%}</h2>
+    <h2>{% trans 'Publishers '%} ({{ publishers|length }})</h2>
+    <details>
```

Comment:
I assume you're aware but this will hide these by default. I think I'm OK with that.

Reply:
Yea -- I don't think its super valuable to take up the first 2 pages of screen, and easy enough to open if needed.

Reply:
@davidfischer I actually want a way to hide the per-publisher breakdown table as well, but `details` doesn't work on tables, so I punted for now :)